types/leaflet-responsive-popup/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for leaflet-responsive-popup 0.6
+// Project: https://github.com/yafred/leaflet-responsive-popup
+// Definitions by: Stan Kurek <https://github.com/stankurek>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import * as L from 'leaflet';
+
+declare module 'leaflet' {
+    interface ResponsivePopupOptions extends PopupOptions {
+        hasTip?: boolean;
+    }
+
+    class responsivePopup extends Popup {
```

Comment:
Looking at https://unpkg.com/leaflet-responsive-popup@0.6.4/leaflet.responsive.popup.js::
`ResponsivePopup` seems like a class and `responsivePopup` seems like a function so you need both, one class and one function

Reply:
Good point. Thanks for taking the time to check it.